src/Notification/Notification.module.scss

```diff
@@ -0,0 +1,107 @@
+@import '../mixins.scss';
+@import '../variables.scss';
+
+.wrapper {
+    @include text-normal;
+
+    border-radius: $border-radius;
+    padding: 14px 18px 16px 15px;
+    width: 320px;
+    margin-bottom: 15px;
+}
+
+.default {
+    color: var(--grey-dark-2);
+    background-color: var(--white);
+    fill: var(--black);
+
+    .title {
+        color: var(--black);
+    }
+
+    .closeIcon {
+        fill: var(--grey-dark);
+    }
+}
+
+.header {
+    display: flex;
+    align-items: center;
+}
+
+.closeIcon {
+    cursor: pointer;
+    width: 13px;
+    height: auto;
+}
+
+.titleIcon {
+    margin: 0 15px 0 0;
+}
+
+.title {
+    font-weight: bold;
+    padding-right: 15px;
+    width: 100%;
+}
+
+.description {
+    margin-top: 8px;
+    padding-left: 31px;
+    padding-right: 25px;
+}
+
+.info {
+    fill: var(--blue);
+    &.solid {
+        background-color: var(--blue);
+    }
+}
+
+.warning {
+    fill: var(--orange);
+    &.solid {
+        background-color: var(--red);
+    }
+}
+
+.error {
+    fill: var(--red);
+    &.solid {
+        background-color: var(--orange);
+    }
+}
+
+.success {
+    fill: var(--green);
+    &.solid {
+        background-color: var(--green);
+    }
+}
+
+.solid {
```

Comment:
Variants should be placed after all default styles, i.e `.solid` should be placed at the very end. 
Also, everything that the variant modifies should be located inside of it:
```scss
// right now
.success {
    &.solid {
    }
}

// better
.solid {
   &.success {
      // styles
   }

   &.error {
      // styles
   }
}
```